Apps/Sandcastle/gallery/Labels.html

```diff
@@ -214,11 +215,55 @@
               Sandcastle.highlight(setRightToLeft);
             },
           },
+          {
+            text: "Animate Label",
+            onselect: function () {
+              animateLabel();
+              Sandcastle.highlight(animateLabel);
+            },
+          },
         ]);
 
         Sandcastle.reset = function () {
           viewer.entities.removeAll();
+          if (Cesium.defined(labelListenerCallback)) {
+            labelListenerCallback();
+          }
         };
+
+        function animateLabel() {
```

Comment:
The first line in `animateLabel` should be `Sandcastle.declare(addToolbarMenu);` so that the line gets highlighted when "Animate Label" is selected from the drop down.